pylint/test/functional/useless-import-alias.py

```diff
@@ -0,0 +1,15 @@
+# pylint: disable=unused-import, missing-docstring, invalid-name, reimported, import-error, wrong-import-order, no-name-in-module, relative-beyond-top-level
+from collections import OrderedDict as OrderedDict # [useless-import-alias]
+from collections import OrderedDict as o_dict
+import os.path as path # [useless-import-alias]
+import os.path as p
+import foo.bar.foobar as foobar # [useless-import-alias]
```

Comment:
I don't find this useless. Let's just keep this check for cases such as `from collections import OrderedDict as OrderedDict` otherwise it might be too noisy for folks

Reply:
I disagree. I think using `from foo.bar import foobar` is more conventional in this case. Could there be a reason why you would have to use `as` instead of `import from`?

Reply:
We discussed this and it's good as is.

Reply:
Yes. No changes are required.